Lesson11/merve/index.html

```diff
@@ -0,0 +1,46 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <nav id="navbar" class="nav">
```

Comment:
Make sure to use <ul> and <li> elements as we did before when creating menus. Also, make sure that every <a> element in the menu points to the correct part of the webpage.